### ISSUE TITLE
1.14 critical plugin error

### ISSUE THREAD

**Issue by baxandrei** [lenis0012/LoginSecurity-2#134]
```
[10:28:47] [pool-4-thread-1/WARN]: Exception in thread "pool-4-thread-1"
[10:28:47] [pool-4-thread-1/WARN]: java.lang.IllegalStateException: AuthActionEvent cannot be triggered asynchronously from another thread.
[10:28:47] [pool-4-thread-1/WARN]:      at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:502)
[10:28:47] [pool-4-thread-1/WARN]:      at com.lenis0012.bukkit.loginsecurity.session.PlayerSession.performAction(PlayerSession.java:160)
[10:28:47] [pool-4-thread-1/WARN]:      at com.lenis0012.bukkit.loginsecurity.session.PlayerSession$2.run(PlayerSession.java:142)
[10:28:47] [pool-4-thread-1/WARN]:      at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[10:28:47] [pool-4-thread-1/WARN]:      at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[10:28:47] [pool-4-thread-1/WARN]:      at java.lang.Thread.run(Thread.java:748)

```

**Comment by lenis0012** [comment]
This issue is indeed related to LoginSecurity, and I will work on fixing it.

**Comment by lenis0012** [comment]
This should be fixed in 2.1.9
